repl/src/test/scala/ammonite/repl/AdvancedTests.scala

```diff
@@ -260,6 +260,17 @@ object AdvancedTests extends TestSuite{
         error: not found: value x
       """)
     }
+    'compilerPlugin{
+      check.session("""
+        @ load.plugin.ivy("org.spire-math" %% "kind-projector" % "0.6.3")
```

Comment:
We probably should also have a negative compile test to make sure stuff loaded via `load.plugin` doesn't end up in the `evalClassloader`.